### PR TITLE
Bump to 0.3.6 & no more syncing for the frontend

### PR DESCRIPTION
**Pull request by TonyGiorgio**
Depends on a new cut of mutiny-wasm.

It's behaving much more fluid now. Excited about this.

### CODE REVIEW

**Comment by benthecarman** [comment]
Probably want to remove the syncing indicator on the on-chain balance too

**Comment by TonyGiorgio** [comment]
> Probably want to remove the syncing indicator on the on-chain balance too. Can make @futurepaul do that tho

Handled that in here.

File: package.json
Comment (futurepaul) [inline package.json:1]: can you bump the mws package version as well. I'm keeping it in sync with the mutiny-wasm version

**Review by futurepaul — CHANGES_REQUESTED** [review]
works on my machine, just would like the version bumped

**Review by futurepaul — APPROVED** [review]

**Pull request merged**